src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/CachedDataAnnotationsModelMetadata.cs

```diff
@@ -326,19 +326,6 @@ protected override bool ComputeShowForEdit()
             }
         }
 
-        private static IEnumerable<string> SplitString(string original)
```

Comment:
not exactly.  it's unused code I'm moving.  destination could be `RemoteAttribute` instead of commonizing it.  sure.  but it should be a move.